admin/schema.php

```diff
@@ -884,6 +884,27 @@
 
 # Release marker: 1.3.0
 
+# Count / occurrences / votes 
+$g_upgrade[210] = array( 'AddColumnSQL', array( db_get_table( 'bug' ), "
+	votes					I		NOTNULL DEFAULT '1'",
```

Comment:
I would separate the concept of reporting a bug from voting for it.  For example, imagine a world where user has a limit of 10 votes to use across open bugs, they may have reported 20 bugs, but put their votes on a subset of these.  We may event decide in the future to enable users to put multiple points on same issue (e.g. user voice enable 1 to 3 votes on a single feature request).

As for the million requests, that is not going to happen because of the max limit (50).  Your point about a million attempts to add is true and we must make sure that doesn't become a denial of service attach on the server, and hence declining new attempts to add reports should be as cheap as possible.

One thing that may be complicating the story is attempting to combine the votes concept and the hit-count concept.  Is that the case?  Should we just focus on the hit-count and use the appropriate terminology and remove ability for a user to do it interactively?  If a user says I want this too or I have this too, then it is not automatic crash reporting anymore and we don't have any info for it, it is just a note or a vote.